CHANGES.rst

```diff
@@ -29,6 +30,7 @@
 - Raise ``NoSuchTableError`` when trying to reflect a table that doesn't exist.
   (`Issue #38 <https://github.com/graingert/redshift_sqlalchemy/issues/38>`_)
 
+
```

Comment:
?

this file is auto-generated by zest.releaser do we need this extra new-line?